tests/test_setup_command.py

```diff
@@ -1,4 +1,27 @@
+import tomlkit
+
+
+def check_result(result, reference, prefix="running pyproject\n"):
+    """
+    Check the result succeeded, and matches the expected output.
+    """
+    assert result.returncode == 0
+    assert result.stdout.startswith(prefix)
+
+    # Parse the reference in case of formatting differences
+    reference_parsed = tomlkit.parse(reference)
+
+    # Parse the resultant text:
+    result_parsed = tomlkit.parse(result.stdout[len(prefix) :])
+
+    # Assert equivalence
+    assert result_parsed == reference_parsed
```

Comment:
I missed this before, but the comments on lines 14 and 17 seem kind of redundant (somewhat along the lines of the proverbial `i++; // increment i`), so I'd suggest removing them, but this is not a big deal and not something I'll insist on. Let me know if you really want to keep these.

```suggestion
    result_parsed = tomlkit.parse(result.stdout[len(prefix) :])

    assert result_parsed == reference_parsed
```

Reply:
Yep, I was sort-of on auto-pilot writing those comments.  They've been removed.